kolibri/plugins/learn/assets/src/state/actions/main.js

```diff
@@ -798,13 +667,12 @@ function closeExam(store) {
 }
 
 export {
+  _contentState,
```

Comment:
I left it because it's a helper action, and I thought the pattern was to include a `_` in that case. I needed it in another file (as a helper), though. Is the pattern actually "use a `_` if it's not going to be exported"?